src/vs/gitpod/browser/workbench/workbench.html

```diff
@@ -28,17 +52,17 @@
 		<!-- Workbench Icon/Manifest/CSS -->
 		<link rel="icon" href="/favicon.ico" type="image/x-icon" />
 		<link rel="manifest" href="/manifest.json" />
```

Comment:
should not we add to all link tags above?

Reply:
If they're interesting to us, let's do it. I figured that they aren't critical so it's not so important.

Reply:
I see, then ok.